### PR TITLE
Feature/analytics

### PR DESCRIPTION
**Pull request by javiermediavilla**
Random Forest training

### CODE REVIEW

File: Analytics.py [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    :param y_test: A numpy array containing the test targets
+    :param n_estimators: The number of estimators
+    :param max_depth: The depth of each tree estimator
+    :param criterion: The criterion for the training process
+    :param visualization: A bool to activate the visualization plots
+    
+    :return RF_model: An object containing the trained edmodel
+    """
+    inicio = time.time()
+    RF_model = sklearn.ensemble.RandomForestRegressor(n_estimators=n_estimators,max_depth=max_depth,criterion=criterion)
```
Comment (mgarciafernandez-altran) [inline Analytics.py:85]: variables shall be all lowercase this shall be named `rf_model`

File: Analytics.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+import matplotlib.pyplot
+import numpy
+import sklearn.ensemble
+import os
+import time
+import typing 
+##################
+
+
+######### Modificable Variables #########
```
Comment (mgarciafernandez-altran) [inline Analytics.py:27]: There shall not be 'hard codded variables at any Python code. These are envirionment variables. Please read them from an external coniguration file, like a YAML

File: Analytics.py [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    First, load the data, and stract the samples and targets for both sets (train and test),
+    also load the varaibles used for the standarization of the data
+    """
+    x_train, x_test, y_train, y_test, mean_standard, std_standard = load_dataset()
+    """
+    Second, the model is trained and the mse performance is measured for the test and
+    training sets. If the visualization variable is set to True, also the function
+    plot a visual example of the prediction
+    """
+    RF_model = trainer_RandomForest(x_train=x_train,
```
Comment (mgarciafernandez-altran) [inline Analytics.py:194]: let the variable be lowercase `rf_model`

File: Analytics.py [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+    x_test = data['x_test']
+    y_train = data['y_train']
+    y_test = data['y_test']
+    mean_standard = data['mean_standarization']
+    std_standard = data['std_standarization']
+    
+    return x_train, x_test, y_train, y_test, mean_standard, std_standard
+    
+
+def trainer_RandomForest(x_train : numpy.ndarray, x_test : numpy.ndarray, y_train : numpy.ndarray, y_test : numpy.ndarray, n_estimators : int, max_depth : int, criterion : str, visualization : bool) -> sklearn.ensemble.RandomForestRegressor:
```
Comment (mgarciafernandez-altran) [inline Analytics.py:65]: function shall be named lowercase `trainer_randomforest`

File: Analytics.py [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+        matplotlib.pyplot.show()
+        
+        print('execution time [seg]  -> ', round(fin-inicio, 4))
+        print('Train MSE             -> ', round(mse_tr, 4))
+        print('Test MSE              -> ', round(mse_ts, 4))
+        
+    return RF_model
+
+
+def save_model_method(RF_model : sklearn.ensemble.RandomForestRegressor, save_model : bool) -> None:
```
Comment (mgarciafernandez-altran) [inline Analytics.py:116]: here `rf_model` is an argument and shall be also lowercase

**Review by mgarciafernandez-altran — CHANGES_REQUESTED** [review]

**Comment by javiermediavilla** [comment]
All changes done.

**Comment by javiermediavilla** [comment]
DONE

**Pull request closed without merging**